raiden/network/pathfinding.py

```diff
@@ -162,6 +157,19 @@ def configure_pfs_or_exit(
         )
         sys.exit(1)
     else:
+        if "payment_address" not in pathfinding_service_info:
```

Comment:
We could only check that when the PFS' fee is higher that 0, but as we always send the address this is not important.